### PR TITLE
-

### PR DESCRIPTION
**Pull request by juanmagal**
Pull request title should be `MF-XXX - description` or `NOISSUE - description` where XXX is ID of issue that this PR relate to.
Please review the [CONTRIBUTING.md](./CONTRIBUTING.md) file for detailed contributing guidelines.

### Which issue(s) does this PR fix/relate to?
Put here `Resolves #XXX` to auto-close the issue that your PR fixes (if such)

### CODE REVIEW

**Review by dborovcanin — CHANGES_REQUESTED** [review]
@juanmagal Thanks for your contribution. Can you, please, read the aforementioned [CONTRIBUTING.md](https://github.com/mainflux/mainflux/blob/master/CONTRIBUTING.md) file and follow the instructions for sending pull requests? You need to add an appropriate PR title (check contribution guide and PR template for naming recommendations), a short explanation of what is the purpose of this PR, and sign off all the commits.

File: k8s/mainflux/users.yml [hunk 1]
```diff
@@ -47,7 +47,7 @@ spec:
     apiVersion: extensions/v1beta1
     kind: Deployment
     name: users
-  minReplicas: 3
+  minReplicas: 1
```
Comment (chombium) [inline k8s/mainflux/users.yml:50]: always cleanup the files after testing ;)

**Review by chombium — CHANGES_REQUESTED** [review]
Hi @juanmagal,

At first thanks and congratulations on sending the first PR to Mainflux as a new contributor. I really like that you've found a possible enhancement and you've start working on it. I also must agree with @dusanb94 and we'll have to help you to clean up the PR before the changes get merged.

I would generally suggest that you:
1. Clean up your commits and get rid of the merge request commits (check the [contribution guide](https://github.com/mainflux/mainflux/blob/master/CONTRIBUTING.md), you should use `git pull --rebase upstream master`

2. You have to sign off  your commits. `git commit -s...`
3. You've copied most of the config files from the docker and k8s folders in the helm folder. This way there is a huge overhead of keeping the files in sync and as the configuration grows, this will turn out in to a maintenance nightmare. The right way to go is to keep the configuration files where they are, so that each of them is a single source of truth and to adjust the helm template (yes I know that it will more complex) :)

@mainflux/contributors this PR refers to #388 

HTH,
Jovan
P.S. keep up the good work

**Comment by juanmagal** [comment]
Hi,

 I am sorry, I pushed the button and didn't want to, so please disregard my
request. I will provide the correct changes once they are ready and they
follow the contribution instructions.

 My apologies for the inconveniences.

Best regards,

Juanma

**Comment by chombium** [comment]
@juanmagal no worries, take all the time you need, to finish the PR.
We don't need to reject the PR, I've added the "work in progress" label to it, so that we know that it's not finished.

Happy hacking

**Comment by manuio** [comment]
@juanmagal Please, edit the name of the PR. Pull request title should be `MF-XXX - description` or `NOISSUE - description` where XXX is ID of issue that this PR relate to.

**Comment by drasko** [comment]
@juanmagal as @manuio pointed out - can you please edit title and description of this PR?

**Comment by juanmagal** [comment]
This pull request was open by mistake, once it is ready I will open a new one. I'm sorry for the inconveniences.

**Pull request closed without merging**